CHANGELOG/v1.6.2.md

```diff
@@ -0,0 +1,34 @@
+## 👌 Kubernetes version support
+
+- Management Cluster: v1.25.x -> v1.29.x
+- Workload Cluster: v1.23.x -> v1.29.x
+
+[More information about version support can be found here](https://cluster-api.sigs.k8s.io/reference/versions.html)
+
+## Highlights
+* :warning: Warning: This release fixes a bug (#10051) that was introduced in v1.6.1, which caused a regression in the conversion of v1alpha3/v1alpha4 objects. It is recommended to upgrade to v1.6.2 to avoid the issue.
```

Comment:
```suggestion
* :warning: Warning: This release fixes a bug (#10051) that was introduced in v1.6.0, which caused a regression in the conversion of v1alpha3/v1alpha4 objects. It is recommended to upgrade to v1.6.2 to avoid the issue.
```